ui/providers/blank/provider.go

```diff
@@ -0,0 +1,15 @@
+package blank
```

Comment:
I wouldn't bother having a `ui/providers` path, and would instead use something like `ui/blankui` and `ui/webui`, since you're unlikely to have anything in here that's *not* a UI provider. It makes the usage of the package more natural (e.g. `webui.New`) and means you don't need to override the name to make it sound less gross.

See `io/ioutil` for an example of this in the stdlib.